edb/edgeql/codegen.py

```diff
@@ -756,6 +756,21 @@ def _ddl_visit_bases(self, node: qlast.BasesMixin) -> None:
             self._write_keywords(' EXTENDING ')
             self.visit_list(node.bases, newlines=False)
 
+    def _ddl_clean_up_commands(
+        self,
+        commands: Sequence[qlast.DDLOperation],
+    ) -> Sequence[qlast.DDLOperation]:
+        # Always omit orig_expr fields from output since we are
+        # using the original expression in TEXT output
+        # already.
+        return [
+            c for c in commands
+            if (
+                not isinstance(c, qlast.SetField)
+                or not c.name.startswith('orig_')
```

Comment:
With the new normalization we no longer need the `orig_` hacks at all.  Would it make sense to clean that up as part of this PR or would you like to do a followup?

Reply:
I would probably make that a separate clean-up PR. Basically we want to land the name normalization and then we can trim this and make further improvements.